wifiphisher/common/victim.py

```diff
@@ -0,0 +1,189 @@
+"""
+This module was made to keep track the information of the victims connected to the rogue AP
+"""
+import os
+from macmatcher import MACMatcher as macmatcher
+
+dir_of_executable = os.path.dirname(__file__)
+path_to_project_root = os.path.abspath(os.path.join(dir_of_executable, '../../wifiphisher'))
+dir_of_data = path_to_project_root + '/data/'
+
+
+path_to_mac_file = dir_of_data + "wifiphisher-mac-prefixes"
+path_to_url_file = dir_of_data + "wifiphiser-os-initial-requests"
```

Comment:
There are constants. They can be defined in `constants.py` file and import them from there.

Reply:
done